test/test_packages/algorithms/test_fogi_gst.py

```diff
@@ -2,7 +2,7 @@
 import numpy as np
 
 import pygsti
-from pygsti.modelpacks import smq1Q_XYI as std
+from pygsti.modelpacks import smq1Q_XY as std
```

Comment:
Was the switch here for speed? I'm for removal of things that don't change the tested functionality and make it faster, but identity for FOGI seems like something where potentially we should explicitly test it?

Reply:
This was primarily for speed. The secondary thinking was that FOGI for the identity should (in principle) be trivial, since all of the parameters of the idle gate are instrinsic FOGI quantities, so this seemed at first glance uninteresting from a testing perspective, at least insofar as we're really only testing the process of fitting models using FOGI and the parameter interposers. That said, the fact that we _know_ all of the parameters of the idle gate are FOGI does also mean this would be a great candidate for a correctness check. We currently aren't doing any correctness checks for FOGI-basis construction (or at least we aren't doing these properly), and since we pretty much know for certain now that the SPAM FOGI quantities are not being constructed correctly at the moment I don't think we should invest the time into adding those checks just yet. When we do, however, doing this check for the XYI model would be smart.

@juangmendoza19, could you add this to your to-do list as part of the FOGI code updates you're working on? This isn't urgent, but we're liable to lose track of this.

Reply:
Noted. I was already planning on having some unit tests as part of the FOGI PR, this seems like a good one to have in there